.github/workflow_scripts/test_eda.sh

```diff
@@ -13,4 +13,8 @@ install_features
 install_eda
 
 cd eda/
-python3 -m pytest --mypy --junitxml=results.xml --runslow "$ADDITIONAL_TEST_ARGS" tests
+python3 -m tox -e lint
```

Comment:
Shall we perform these checks for all modules

Reply:
yes, this was done as the first POC, I will work in a background to migrate other modules to tox.